vendor/vendor.json

```diff
@@ -692,68 +692,54 @@
 			"versionExact": "master"
 		},
 		{
-			"checksumSHA1": "FaDuiEL1h9wdNYBG5uyjvlBDPEY=",
+			"checksumSHA1": "WOLMxJCYaKYleZvmGEClhuhM3BE=",
 			"path": "github.com/hyperledger/fabric/common/metrics",
-			"revision": "7891c890ab6ca541b2184a32f52d05f2f964adcf",
-			"revisionTime": "2019-10-28T13:05:25Z",
+			"revision": "7917a40ff6f694b34c971b1f8285de8b1ac39777",
```

Comment:
@saad-karim  - why are we doing this?   I had previously updated the bccsp dependency to use a revision from the master branch so that we could use the latest from metrics on the master branch as well ... I don't think we need/want this change

Reply:
@mastersingh24 The reasoning behind this is that in ibp-operator when we vendor 1.4.x Fabric, we run into issues when vendoring 1.4.x CA. The issue being that in 1.4.x Fabric there is no field `LabelHelp` as part of the metric structs. We want to vendor the same version of CA and Fabric ,so that is why we are trying to sync up the versions.

Reply:
@saad-karim 
But fabric-ca 1.4 also vendors bccsp from fabric master as well .... we specifically made that change because we were vendoring metrics from fabric master as well for the LabelHelp fix which was not backported.   I think there is much more to consider here

Reply:
Right, fabric-ca 1.4 does vendor bccsp from fabric master. This means that when we vendor dependencies in operator we have 1.4 fabric-ca and then master fabric. Ideally, we wanted to vendor the same version of fabric and fabric-ca.